source/util/string_utils.h

```diff
@@ -44,9 +46,9 @@ std::string CardinalToOrdinal(size_t cardinal);
 // string will be empty.
 std::pair<std::string, std::string> SplitFlagArgs(const std::string& flag);
 
-// Encodes a string as a sequence of words, using the SPIR-V encoding.
-inline std::vector<uint32_t> MakeVector(std::string input) {
-  std::vector<uint32_t> result;
+// Encodes a string as a sequence of words, using the SPIR-V encoding, appending
+// to an existing vector.
+inline void AppendToVector(std::string input, std::vector<uint32_t>& result) {
```

Comment:
Please avoid non-const references.
This codebase has avoided them (except for I/O streams).